app/extensions.js

```diff
@@ -137,15 +151,56 @@ let generateBraveManifest = () => {
     cspDirectives['style-src'] = '\'self\' \'unsafe-inline\' http://' + devServer
   }
 
-  var csp = ''
-  for (var directive in cspDirectives) {
-    csp += directive + ' ' + cspDirectives[directive] + '; '
-  }
-  baseManifest.content_security_policy = csp.trim()
+  baseManifest.content_security_policy = concatCSP(cspDirectives)
 
   return baseManifest
 }
 
+// Returns the Chromium extension manifest for the torrentExtension
+// The torrentExtension handles magnet: URLs
+// Analagous to the PDFJS extension, it shows a special UI for that type of resource
+let generateTorrentManifest = () => {
+  let cspDirectives = {
+    'default-src': '\'self\'',
+    'frame-src': 'http://localhost:*',
+    'form-action': '\'none\'',
+    'referrer': 'no-referrer',
+    'style-src': '\'self\' \'unsafe-inline\'',
+    'img-src': '* data: blob:'
+  }
+
+  // TODO:
+  // * Move WebTorrent to its own process, similar to the way it's done in WebTorrent Desktop
+  // * Remove this CSP exception:
+  cspDirectives['connect-src'] = '*'
```

Comment:
This was only needed when `webtorrent` was running in a `<script>` tag in the chrome extension, so it could connect to http web seeds, and websocket servers. Not needed anymore